lib/matplotlib/tests/test_font_manager.py

```diff
@@ -268,6 +270,27 @@ def test_fontcache_thread_safe():
                     f"{proc.returncode}.")
 
 
+def test_fontentry_dataclass():
+    fontent = FontEntry(name='font-name')
+
+    assert type(fontent.__doc__) == str
+
+    png = fontent._repr_png_()
+    html = fontent._repr_html_()
```

Comment:
Move to the respective places of usage. That's simpler to understand than defining stuff up front and only use it later.

Reply:
Done!